adapters/zwave/zwave-classifier.js

```diff
@@ -46,7 +47,28 @@ class ZWaveClassifier {
             'name': 'toggle',
             'description': 'Toggles On/Off',
         }];
-        node.propertyMap = {'on': valueId };
+        node.propertyMaps = {'on': valueId };
+        node.values.on = node.zwValues[valueId].value;
+
+        if (node.zwInfo.productId == 0x0060 &&
+            (node.zwInfo.productType & 0xff) == 0x03) {
```

Comment:
I have no issues with (and agree with) having some type of generic mechanism. Do you want me to remove this? Or leave it in until the generic mechanism is implemented?